pex/bin/pex.py

```diff
@@ -503,11 +503,18 @@ def make_relative_to_root(path):
   return os.path.normpath(path.format(pex_root=ENV.PEX_ROOT))
 
 
-def main():
+def main(args=None, log_callback=None):
+  if log_callback is not None:
```

Comment:
not crazy about this bit, how about just:

```
def main(args, log=log, exiter=sys.exit):
...
  exiter(pex.run(...))
...
if __name__ == '__main__':
  main(sys.argv[1:])
```

and then eliminating all of the `if log_callback` & `if args` checks + the `raise SystemExit`?